src/newViews/UserGroup/index.tsx

```diff
@@ -0,0 +1,279 @@
+    description: string;
+    role: string;
+    memberships: Membership[];
+    globalCrisisMonitoring: boolean;
+    createdAt: string;
+    modifiedAt: string;
+}
+
+interface UsergroupValueToSend {
+    title: string;
+    description?: string;
+    globalCrisisMonitoring?: boolean;
+}
+
+const MAX_ITEMS_PER_PAGE = 10;
+const usergroupKeySelector = (d:Usergroup) => d.id;
+
+interface Props {
+    activeUser: { userId: number };
+}
+
+function UserGroup(props: Props) {
+    const {
+        activeUser,
+    } = props;
+
+    const [activePage, setActivePage] = useState<number>(1);
+    const [
+        showAddUSerGroupModal,
+        setModalShow,
+        setModalHidden,
+    ] = useModalState(false);
+
+    const usergroupQuery = useMemo(() => ({
+        user: activeUser.userId,
+        offset: (activePage - 1) * MAX_ITEMS_PER_PAGE,
+        limit: MAX_ITEMS_PER_PAGE,
+    }), [activeUser.userId, activePage]);
+
+    const {
+        pending: usergroupGetPending,
+        response: usergroupResponse,
+    } = useRequest<MultiResponse<Usergroup>>({
+        url: 'server://user-groups/member-of/',
+        method: 'GET',
+        query: usergroupQuery,
+    });
+
+    const {
+        pending: postNewUsergroupPending,
+    } = useLazyRequest<unknown, UsergroupValueToSend>({
+        url: 'server://user-groups/',
+        method: 'POST',
+    });
+
+    console.warn(postNewUsergroupPending);
+
+    // FIXME: To be handled
+    const handleEditUsergroupClick = useCallback(() => {}, []);
+    const handleDeleteUsergroupClick = useCallback(() => {}, []);
+
+    // FIXME: To be handled
+    const handleEditMemberClick = useCallback(() => {}, []);
+    const handleDeleteMemberClick = useCallback(() => {}, []);
+
+    const columns = useMemo(() => {
+        const actionColumn: TableColumn<
+            Usergroup,
+            number,
+            ActionCellProps<number>,
+            TableHeaderCellProps
+        > = {
+            id: 'action',
+            title: _ts('usergroup', 'actionLabel'),
+            headerCellRenderer: TableHeaderCell,
+            headerCellRendererParams: {
+                sortable: false,
+            },
+            cellRenderer: ActionCell,
+            cellRendererParams: passedUserId => ({
+                itemKey: passedUserId,
+                onEditClick: handleEditUsergroupClick,
+                onDeleteClick: handleDeleteUsergroupClick,
+                editButtonTitle: 'Edit',
+                deleteButtonTitle: 'Delete',
+                deleteConfirmationMessage: 'Really delete',
+            }),
+        };
+
+        return ([
+            createStringColumn<Usergroup, number>(
+                'group',
+                _ts('usergroup', 'groupLabel'),
+                item => item.title,
+            ),
+            createStringColumn<Usergroup, number>(
+                'members',
+                _ts('usergroup', 'membersLabel'),
+                item => item.memberships.length.toString(),
```

Comment:
This should come from the server as `membershipCount` instead of fetching the length.
Could you create a seprate issue to add `membershipCount` in current user group API?
And also create a a seraparte issue to add a separate paginated API for fetching user group memberships.

Reply:
Okay, sure!